builtin/providers/openstack/resource_openstack_compute_floatingip_associate_v2.go

```diff
@@ -90,6 +90,22 @@ func resourceComputeFloatingIPAssociateV2Read(d *schema.ResourceData, meta inter
 	d.Set("fixed_ip", fixedIP)
 	d.Set("region", GetRegion(d))
 
+	// check whether floatingIP or associated instance deleted
+	config := meta.(*Config)
+	computeClient, err := config.computeV2Client(GetRegion(d))
+	if err != nil {
+		return fmt.Errorf("Error creating OpenStack compute client: %s", err)
+	}
```

Comment:
Can you move lines 94-98 to the top of the function like how it is in the other functions?